packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
 import throttle from 'lodash/throttle';
-import { View, StyleSheet, Pressable } from 'react-native';
+import { View, StyleSheet, Pressable, Platform } from 'react-native';
 import Animated from 'react-native-reanimated';
 import PagerView from 'react-native-pager-view';
 
 import type { UICarouselViewContainerProps, UICarouselViewPageProps } from '../../types';
 import { usePageStyle } from '../animations';
+import { Pagination } from '../Pagination';
 
 const returnPages = (
     pages: React.ReactElement<UICarouselViewPageProps>[],
     shouldPageMoveOnPress: boolean,
     nextPage: (index: number) => void,
 ) => {
     const onPress = React.useCallback(
-        throttle((index: number) => nextPage(index), 250, { leading: false, trailing: true }),
+        throttle((index: number) => nextPage(index), 1000, { leading: true, trailing: true }),
         [],
     );
 
     return pages.map((page, index) => {
-        const onPressPage = () => onPress(index);
+        const onPressPage = () => {
+            if (Platform.OS === 'android') {
+                nextPage(index);
+            } else {
+                onPress(index);
+            }
+        };
```

Comment:
```suggestion
        const onPressPage = Platform.select({ android: nextPage, default: onPress });
```

That way you won't create additional wrapper function :)